lib/bosh/NetworkSegmentIndex.js

```diff
@@ -48,8 +47,8 @@ NetworkSegmentIndex.capacity = calculateCapacity;
 
 module.exports = NetworkSegmentIndex;
 
-function calculateCapacity(subnet) {
-  let infrastructure = config.director.infrastructure;
+function calculateCapacity(subnet, config) {
+  let infrastructure = config.infrastructure;
```

Comment:
Infrastructure could be different for different directors ? 
so we cannot arbitrarily decide which infra to return ?